frontend/packages/erd-core/src/stores/schema/SchemaProvider.tsx

```diff
@@ -23,13 +23,16 @@ export const SchemaProvider: FC<Props> = ({ children, current, previous }) => {
       tables: {},
     }
     const diffItems = buildSchemaDiff(previous ?? emptySchema, current)
+    const filteredDiffItems = diffItems.filter(
+      (item) => item.status !== 'unchanged',
+    )
```

Comment:
I could always filter it out in buildSchemaDiff if it's not needed, but once it's done, I guess I'll just leave it as is.